<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataManager</class>
 <widget class="QMainWindow" name="DataManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Manager</string>
  </property>
  <widget class="QWidget" name="cWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="vLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="topLayout">
        <item>
         <layout class="QVBoxLayout" name="leftListLayout">
          <item>
           <widget class="QListView" name="stack_list"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="bottomLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="usernameEdit">
          <property name="text">
           <string>anon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonParams">
          <property name="text">
           <string>Parameter Settings</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonQuit">
          <property name="text">
           <string>Quit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
